addons/html_builder/static/src/website_preview/website_builder_action.js

```diff
@@ -46,6 +46,11 @@ export class WebsiteBuilder extends Component {
             // You can't wait for rendering because the Builder depends on the page style synchronously.
             effect(
                 (websiteContext) => {
+                    // this is needed to avoid error when we try to change the mode after we have been 
+                    // redirected to the website after creating a new page/content
+                    if (!this.websitePreviewRef.el) {
+                        return;
+                    }
```

Comment:
if you don t read websiteContext.isMobile. You will not subscribe anymore to isMobile.